src/stylesheets/components/_header.scss

```diff
@@ -0,0 +1,602 @@
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-font-smoothing: antialiased;
+
+  &:hover {
+    span {
+      @include media($nav-width) {
+        @include nav-border;
+      }
+    }
+  }
+}
+
+.usa-nav-container {
+  @include media($nav-width) {
+    @include outer-container();
+    @include padding(null $site-margins);
+    max-width: $site-max-width;
+  }
+}
+
+.usa-nav {
+  $sliding-panel-width: 26rem;
+  @include padding(2rem);
+  @include position(fixed, 0 0 0 auto);
+  @include size($sliding-panel-width 100%);
+  @include transform(translateX($sliding-panel-width));
+  background: $color-white;
+  border-left: 1px solid $color-gray-light;
+  border-right: 0;
+  display: flex;
+  flex-direction: column;
+  overflow-y: auto;
+  -webkit-overflow-scrolling: touch;
+  z-index: 99999;
+
+  @include media($nav-width) {
+    @include padding(6rem 0 0 null);
+    @include transform(translateX(0));
+    border-left: none;
+    display: block;
+    float: right;
+    overflow-y: visible;
+    position: relative;
+    width: auto;
+  }
+
+  &.is-visible {
+    @include transform(translateX(0));
+    @include transition(all 0.3s ease-in-out);
+  }
+
+  nav {
+    margin-top: 6rem;
+    min-height: 100%;
+
+    @include media($nav-width) {
+      margin-top: 0;
+    }
+  }
+
+  .usa-current {
+    border-left: 4px solid $color-primary;
+    color: $color-primary;
+    font-weight: $font-bold;
+    padding-left: 1.4rem;
+  }
+
+  .usa-button {
+    width: 100%;
+  }
+
+  .usa-search {
+    @include media($nav-width) {
+      margin-left: 1.5rem;
+    }
+  }
+}
+
+.usa-nav-primary {
+  @include usa-sidenav-list;
+  order: 2;
+
+  @include media($nav-width) {
+    display: inline;
+  }
+
+  li {
+    @include media($nav-width) {
+      border-top: none;
+    }
+  }
+
+  > li {
+    width: auto;
+
+    @include media($nav-width) {
+      display: inline-block;
+      margin-left: -4px;
+    }
+
+    > a {
+      @include media($nav-width) {
+        @include padding(1.3rem 1.5rem 1.7rem 1.5rem);
+        color: $color-gray;
+        font-size: $h5-font-size;
+        font-weight: $font-bold;
+        line-height: 1.2;
+      }
+
+      &:hover {
+        @include media($nav-width) {
+          background-color: transparent;
+        }
+      }
+    }
+  }
+
+  button {
+    @include button-unstyled;
+    font-weight: $font-normal;
+    padding: 1.4rem 1.5rem 1rem 1.8rem;
+    -moz-osx-font-smoothing: grayscale;
+    -webkit-font-smoothing: antialiased;
+
+    @include media($nav-width) {
+      @include padding(null 3rem 1.9rem 1.5rem);
+      color: $color-gray;
+      font-size: $h5-font-size;
+      font-weight: $font-bold;
+      width: initial;
+    }
+
+    &:focus,
+    &:active {
+      box-shadow: $focus-shadow;
+    }
+
+    &:hover {
+      background-color: $color-gray-lightest;
+      color: $color-primary;
+
+      @include media($nav-width) {
+        background-color: transparent;
+      }
+    }
+
+    &[aria-expanded=false] {
```

Comment:
Man of these rules seem like repeats from accordion. Do you think we should try and put some of it in a shared place to avoid repetition? Or is the code not similar enough?